homeassistant/components/ovo_energy/sensor.py

```diff
@@ -46,15 +46,15 @@ class OVOEnergySensorEntityDescription(SensorEntityDescription):
         key="last_electricity_reading",
         translation_key="last_electricity_reading",
         device_class=SensorDeviceClass.ENERGY,
-        state_class=SensorStateClass.TOTAL_INCREASING,
+        state_class=SensorStateClass.TOTAL,
```

Comment:
I don't think this one should be changed.
```suggestion
        state_class=SensorStateClass.TOTAL_INCREASING,
```

Reply:
You're right, these are the monetary values